Server/fixtures/mobile.js

```diff
@@ -103,16 +138,6 @@ exports.createFixtures = async () => {
           password: bcrypt.hashSync("admin", 12),
           mailVerification: true,
           notificationsId: notification1.id,
-          itemOpinions: {
```

Comment:
put back the fixtures for the mobile

Reply:
how is this resolved ? where is the itemOpinion ?

Reply:
I put back